public/locales/cs/translation.json

```diff
@@ -181,10 +181,13 @@
 		"Setup has not been completed": "Nastavení nebylo dokončeno"
 	},
 	"WebAuthnScreen": {
+		"Something went wrong, can't retrieve authenticators": "Něco je špatně, nelze načíst autentizátory",
```

Comment:
@Aringoaway `autentikátory` - you did not change it everywhere as I asked you in previous review, so please do now